src/js/components/dashboard/graph/active/SignificanceCircle.jsx

```diff
@@ -47,6 +47,7 @@ export default class SignificanceCircle extends React.Component {
     mouseEntered() {
         this.props.showTooltip({
             ...this.props,
+            itemWidth: 23 * 2,
```

Comment:
I understand that this needs to be a magic number for whatever reason, but do we need the math? Can it just be 46? If it's a number that's used more than once in this file then maybe make it a constant at the top so you can multiply that.